datasets/ua/nabc_sanctions/crawler.py

```diff
@@ -57,13 +58,9 @@ def clean_row(row: Dict[str, Any]) -> Dict[str, Any]:
     return data
 
 
-def parse_date(date):
-    if date is not None:
-        date = date.replace(" ", "")
-    dates = set()
-    for part in h.multi_split(date, [",", "\n", ";"]):
-        dates.update(h.parse_date(part, ["%d.%m.%Y", "dd.%m.%Y"]))
```

Comment:
I think we also need to add these format strings to the metadata `dates.formats` or something